### PR TITLE
Replace personal_values cypress test with equivalent pytest

### LINKED ISSUE #399: Replace cypress personal_values.cy.js with PyTest integration tests.

### PR DESCRIPTION
**Pull request by brianpeiris**
### Detailed information:

- Deleted `personal_values.cy.js` and replaced it with an equivalent pytest in `app/personal_values/tests/test_personal_values_routes.py`
- Removed obsolete `personalValuesEndpoint` cypress command.

### Closing issues: 

List all issues the pull request solve: 

- Closes #399

### Test plan (required)

- [x] The PR contains new PyTest unit/integration tests for any function or functional added. 
- [x] The PR changes existing PyTest unit/integration tests to keep all tests up to date.
- [x] The PR does not lead to degradation in unit test coverage.
- [x] Project parts affected by changes in this PR was tested manually on your local (using Postman or in any other way). List everything you've tested below:
  - Ran all pytests
  - Ran all cypress tests

### CODE REVIEW

File: app/personal_values/tests/test_personal_values_routes.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        assert "id" in personal_value
+        assert "description" in personal_value
+        assert "shortDescription" in personal_value
+        assert "name" in personal_value
+
+    for value_score in json["valueScores"]:
+        assert "personalValue" in value_score
+        assert type(value_score["score"]) == float
+        assert value_score["score"] >= 0
+        assert value_score["score"] <= 10
```
Comment (brianpeiris) [inline app/personal_values/tests/test_personal_values_routes.py:40]: Note that the cypress version had two tests, where the second one used `postScoresSetTwo.json`. But as far as I can tell, the second test didn't really cover more code, and was basically identical to the first test, so I didn't duplicate the test here.
